macpkg/scripts/postinstall

```diff
@@ -1,15 +1,23 @@
 #!/usr/bin/env bash
 EXE_NAME="aws"
 COMPLETER_EXE_NAME="aws_completer"
-sudo mkdir -p /usr/local/bin
-# Files created here are not stored in the bill of materials for the pkg.
-# This means we cannot find and remove them during an installation. We add a
-# custom .install-metadata file here which is read and used by our
-# scripts/installers/uninstall-mac-pkg to clean up any extra files created
-# here. If any new files are added, be sure to add them to the .install-metadata
-# file as well.
-sudo echo "/usr/local/aws-cli/.install-metadata" >> /usr/local/aws-cli/.install-metadata
```

Comment:
Did we want to delete the `.install-metadata` file? This could be helpful when we add support for deleting installations via the installers.